cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/connector/SampleRequest.java

```diff
@@ -113,7 +121,7 @@ public Builder setProperties(Map<String, String> properties) {
     }
 
     public SampleRequest build() {
-      return new SampleRequest(path, limit, properties);
```

Comment:
Even though it's not used now, please add timeout to the builder for completeness

Reply:
Done!